www/nodejs-project/index.js

```diff
@@ -154,6 +156,7 @@ app.post('/load', async function (request, response) {
     console.log("Loading a datArchive")
     var datUrl = request.body.datUrl;
     let key = datUrl.replace('dat://', '')
+    await axios.get(`http://${hexTo32.encode(key)}.lvh.me:3000/dat.json`)
```

Comment:
great you're using lvh.me - I replace the use of localhost:3000 earlier with lvh.me:3000 on Android and IOS for addressing the server and seems to be good.